postgres/datadog_checks/postgres/statements.py

```diff
@@ -27,15 +28,21 @@
 except ImportError:
     from ..stubs import datadog_agent
 
+QUERYID_TO_CALLS_QUERY = """
+SELECT queryid, calls
+  FROM {pg_stat_statements_view}
+  WHERE queryid IS NOT NULL
+"""
+
 STATEMENTS_QUERY = """
 SELECT {cols}
   FROM {pg_stat_statements_view} as pg_stat_statements
   LEFT JOIN pg_roles
          ON pg_stat_statements.userid = pg_roles.oid
   LEFT JOIN pg_database
          ON pg_stat_statements.dbid = pg_database.oid
-  WHERE query != '<insufficient privilege>'
   AND query NOT LIKE 'EXPLAIN %%'
+  AND queryid = ANY('{{ {called_queryids} }}'::bigint[])
```

Comment:
Query is now invalid, you need to replace the first AND with WHERE

Reply:
Somehow the tests were passing with this? 🤔 But I changed it to use WHERE.